app/assets/javascripts/details/shell/action_bar_bridge.js

```diff
@@ -0,0 +1,116 @@
+var ActionBarBridge;
+ActionBarBridge = (function() {
+
+  function ActionBarBridge(c2, config){
+    this.c2 = c2;
+    config = config || {};
+    this.config = {
+      actionBar:      '#action-bar-wrapper',
+      detailsSave:    '#request-details-card',
+      notifications:  '#action-bar-status',
+      editMode:       '#mode-parent',
+      modalCard:      '#modal-wrapper',
+      updateView:     '#mode-parent'
+    }
+    this._overrideTestConfig(config);
+    this._blastOff();
+  }
+
+  ActionBarBridge.prototype._blastOff = function(){
```

Comment:
@dogweather Its a standard from the modular `backbone.js` library `human.js`.

Link: https://github.com/HenrikJoreteg/humanjs

I didnt come up with the syntax. 

Theres a whole book outlining the writing style and naming pattern: http://read.humanjavascript.com/